arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/extensions/setk/at/SetKAt.kt

```diff
@@ -28,7 +28,7 @@ internal val at_singleton: SetKAt<Any?> = object : SetKAt<Any?> {}
   "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
   ReplaceWith(
     "this compose At.set<A>().at(i)",
-    "arrow.optics.set", "arrow.optics.typeclasses.At", "arrow.optics.compose"
+    "arrow.optics.typeclasses.At", "arrow.optics.compose"
```

Comment:
There is no "arrow.optics.compose" import. This needs to be removed from all `ReplaceWith`, this was here already before this PR but let's clean it up here.